### ISSUE TITLE
Question about Experimental Evaluation of  ACM and DBLP Datasets

### ISSUE THREAD

**Issue by FinchNie** [Shen-Lab/GDA-SpecReg#1]
Dear author,
I would like to express my appreciation for your outstanding work. I have a question regarding the experimental evaluation of node classification in the ACM and DBLP datasets. I noticed that a validation set does not appear to have been used, and instead, the best test performance of ACM and DBLP was directly selected. I was wondering if this approach might have some potential limitations or if there is a specific reason for not using a validation set. I would greatly appreciate your clarification on this matter. Looking forward to your reply. 
Thank you.

**Comment by yyou1996** [comment]
Hi @FinchNie,

So sorry for the delayed reply! The metric in ACM/DBLP is validation accuracy without test sets, which is consistent with the prior work (https://github.com/GRAND-Lab/UDAGCN where I gather the benchmark) for fair comparison.

Beyond ACM/DBLP, all other experiments are evaluated with test performance with traditional train/val/test splits.
